german/Singen/Walen/walen.md

```diff
@@ -0,0 +1,8 @@
+Sie denken, dass Sie einer Freunde diese Walen sein können und sie werden Ihnen helfen, wie Sie auf die [Nat Geo](http://www.nationalgeographic.com/) sah.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.